### PR TITLE
added docs for vmui in victorialogs

### PR DESCRIPTION
**Pull request by Amper**

### CODE REVIEW

File: docs/VictoriaLogs/querying/README.md [hunk 1]
```diff
@@ -48,6 +48,28 @@ curl http://localhost:9428/select/logsql/query -H 'AccountID: 12' -H 'ProjectID:
 The number of requests to `/select/logsql/query` can be [monitored](https://docs.victoriametrics.com/VictoriaLogs/#monitoring)
 with `vl_http_requests_total{path="/select/logsql/query"}` metric.
 
+## VMUI
+
+VictoriaLogs provides simple UI for logs query and exploration. The UI is available at `http://victorialogs:8428/vmui`.
```
Comment (tenmozes) [inline docs/VictoriaLogs/querying/README.md:53]: wrong port

File: docs/VictoriaLogs/querying/README.md [hunk 2]
```diff
@@ -48,6 +48,28 @@ curl http://localhost:9428/select/logsql/query -H 'AccountID: 12' -H 'ProjectID:
...
+The UI allows exploring query results via tables or JSON-result:
+
+<img src="vmui.png" width="800" />
```
Comment (tenmozes) [inline docs/VictoriaLogs/querying/README.md:56]: image is missing in PR

File: docs/VictoriaLogs/querying/README.md (hunk 1 above)
Comment (Amper) [reply docs/VictoriaLogs/querying/README.md]: fixed

File: docs/VictoriaLogs/QuickStart.md [hunk 3]
```diff
@@ -96,4 +96,11 @@ as demo for running fluentbit in kubernetes with VictoriaLogs:
 
 ## How to query logs in VictoriaLogs
 
-See details in [these docs](https://docs.victoriametrics.com/VictoriaLogs/querying).
+You can query logs with [LogsQL](https://docs.victoriametrics.com/VictoriaLogs/LogsQL.html) using the following ways:
```
Comment (tenmozes) [inline docs/VictoriaLogs/QuickStart.md:99]: the following options

File: docs/VictoriaLogs/querying/README.md [hunk 4]
```diff
@@ -48,6 +48,28 @@ curl http://localhost:9428/select/logsql/query -H 'AccountID: 12' -H 'ProjectID:
 The number of requests to `/select/logsql/query` can be [monitored](https://docs.victoriametrics.com/VictoriaLogs/#monitoring)
 with `vl_http_requests_total{path="/select/logsql/query"}` metric.
 
+## VMUI
+
+VictoriaLogs provides simple UI for logs query and exploration. The UI is available at `http://victorialogs:9428/vmui`.
```
Comment (tenmozes) [inline docs/VictoriaLogs/querying/README.md:53]: a simple

File: docs/VictoriaLogs/querying/README.md [hunk 5]
```diff
@@ -48,6 +48,28 @@ curl http://localhost:9428/select/logsql/query -H 'AccountID: 12' -H 'ProjectID:
...
+
+<img src="vmui.png" width="800" />
+
+There are three modes of displaying query results:
+
+- `Group` - results are displayed as a table with rows grouped by stream and fields for filtering.
+- `Table` - displays query results as a table.
+- `JSON` - displays raw JSON response from `/select/logsql/query` endpoint.
+
+This is the first version that has minimal functionality. This version has the following limitations:
```
Comment (tenmozes) [inline docs/VictoriaLogs/querying/README.md:64]: This version has the following limitations: -> 
It comes with the following limitations:

File: docs/VictoriaLogs/querying/README.md [hunk 6]
```diff
@@ -48,6 +48,28 @@ curl http://localhost:9428/select/logsql/query -H 'AccountID: 12' -H 'ProjectID:
...
+
+- The number of query results is always limited to 1000 lines.
+- A tenant 0 is always used in queries.
+
+These limitations will be removed in future versions.
```
Comment (tenmozes) [inline docs/VictoriaLogs/querying/README.md:69]: should we create any bugs?

File: docs/VictoriaLogs/querying/README.md (hunk 4 above)
Comment (Amper) [reply docs/VictoriaLogs/querying/README.md]: fixed

**Review by tenmozes — APPROVED** [review]

**Pull request merged**